### PR TITLE
adding spring controller changes

### PR DESCRIPTION
**Pull request by prashant1067**

### CODE REVIEW

**Comment by prashant1067** [comment]
adding spring controller

File: spring_controller/src/main/java/com/test/controller/TestController.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+/**
+ * @author Prashant Dutta
+ *
+ */
+package com.test.controller;
```
Comment (pivovarit) [inline spring_controller/src/main/java/com/test/controller/TestController.java]: Package name should be com.baeldung

**Comment by pivovarit** [comment]
Please check if this code can't be placed in an already existing module.

**Comment by pivovarit** [comment]
Also, please make short runnable tests for this. There is some Spring MVC testing tools, so this should be fairly easy

**Comment by prashant1067** [comment]
Changed the package name and introduced from com.test.* to com.baledung.*

**Comment by prashant1067** [comment]
Can't find a suitable module to move this code . Since this code is solely linked to controllers I believe that there should be a seperate module for this. Please do  advise on this.

**Comment by pivovarit** [comment]
If spring-all is not suitable, you can start a new one 'spring-controller'

**Comment by prashant1067** [comment]
I have done that.

**Comment by pivovarit** [comment]
And when can we expect tests for this? mockMVC will be helpful

**Comment by prashant1067** [comment]
Added test cases using mock mvc.

**Pull request merged**